corehq/apps/app_manager/views/modules.py

```diff
@@ -1753,6 +1757,16 @@ def get(self, request, domain):
         })
 
 
+class AllCaseTypesView(LoginAndDomainMixin, View):
```

Comment:
nit: generally i approach data endpoints with functional views rather than class-based views

Reply:
nm: i see you were copying the existing pattern above. I think it's fine to leave as is then